sklearn/feature_selection/_rfe.py

```diff
@@ -696,7 +716,27 @@ def fit(self, X, y, groups=None):
         self.estimator_ = clone(self.estimator)
         self.estimator_.fit(self.transform(X), y)
 
-        # Fixing a normalization error, n is equal to get_n_splits(X, y) - 1
-        # here, the scores are normalized by get_n_splits(X, y)
-        self.grid_scores_ = scores[::-1] / cv.get_n_splits(X, y, groups)
+        # reverse to stay consistent with before
+        scores_rev = scores[:, ::-1]
+        self.cv_results_ = {}
+        self.cv_results_["mean_score"] = np.mean(scores_rev, axis=0)
+        self.cv_results_["std_score"] = np.std(scores_rev, axis=0)
+
+        for i in range(scores.shape[0]):
+            self.cv_results_[f"split{i}_score"] = scores_rev[i]
+
         return self
+
+    # TODO: Remove in v1.2 when grid_scores_ is removed
+    # mypy error: Decorated property not supported
+    @deprecated(  # type: ignore
+        "The grid_scores_ attribute is deprecated in version 1.0 in favor "
+        "of cv_results_ and will be removed in version 1.2."
```

Comment:
I think that this will solve the failure

```suggestion
        "The `grid_scores_` attribute is deprecated in version 1.0 in favor "
        "of `cv_results_` and will be removed in version 1.2."
```